getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -326,6 +397,158 @@ are constants:
         FIRE,
     }
 
+
+Code order
+----------
+
+This first section focuses on code order. For formatting, see
+:ref:`code_formatting`. For naming conventions, see :ref:`naming_conventions`.
+
+We suggest to organize GDScript code this way:
+
+::
+
+    01. class_name
+    02. extends
+    03. # docstring
+
+    04. signals
+    05. enums
+    06. constants
+    07. exported variables
+    08. public variables
+    09. private variables
+    10. onready variables
```

Comment:
why not before private variables?

Reply:
As discussed in the issue, several persons want it as close as possible to the _ready function, as onready is evaluated right before _ready and after other properties. But still grouped with the other variables as it's part of the class's properties.

Reply:
ok